### PR TITLE
Update ArtifactOntology.ttl

### PR DESCRIPTION
**Pull request by neilotte**
Addressing https://github.com/CommonCoreOntology/CommonCoreOntologies/issues/160 . 

I performed a scan of all IBEs that are said to bear some subrelation of aboutness and revised the wording.

### CODE REVIEW

**Comment by neilotte** [comment]
@cameronmore  Mind taking a look at this one when you have a moment?

File: ArtifactOntology.ttl [hunk 1]
```diff
@@ -595,7 +595,8 @@ cco:Certificate rdf:type owl:Class ;
 ###  http://www.ontologyrepository.com/CommonCoreOntologies/Chart
 cco:Chart rdf:type owl:Class ;
           rdfs:subClassOf cco:Image ;
-          cco:definition "An Image that is designed to represent an Information Content Entity by means of Written Symbols in order to convey that information in a readily understandable format."@en ;
+          cco:definition "An Image that bears a table, lists, diagrams, or graphs."@en ;
```
Comment (cameronmore) [inline ArtifactOntology.ttl:598]: The previous definition was not clear because it did not define 'written symbol' (which are distinguished from non-symbolic semantic glyphs like letters), however I am always hesitant to enumerate types in a definition, especially since table, diagram, and graph are not currently defined. Also, the items listed should be all singular and not mixed.

**Review by cameronmore — DISMISSED** [review]
Agree but hesitant about the definition of chart.

**Comment by alanruttenberg** [comment]
This shouldn't be done separately from the IBE/ICE refactor, which is being worked on in the CCO weekly grad student weekly, in particular by at least @avsculley and myself in the https://github.com/CommonCoreOntology/CommonCoreOntologies/tree/ice-iba-refactor. Let's coordinate.
It doesn't really deconflict in that e.g/ Chart probably shouldn't be in IBE at all.

File: ArtifactOntology.ttl (hunk 1 above)
Comment (neilotte) [reply ArtifactOntology.ttl:598]: @cameronmore I guess in my mind the question is a) is this a better definition than the present one? Answer: yes. b) Is there a readily available one that is better? Answer: I don't know. Can you think of one? :)

**Comment by neilotte** [comment]
@alanruttenberg  I was executing here only to update the definitions and not the broader of issue of what should and shouldn't be an IBE. Good to know you've been tackling this more comprehensively; however, I see your branch is now 35 commits behind (can you and @avsculley pull from development? Also, are @APCox @johnbeve  and @mark-jensen  tracking this refactor?

**Comment by alanruttenberg** [comment]
I believe they are aware. It is a WIP, but if not they are now :-) 
Agree it should pull to update. But it should be noted that it isn't exclusive to us (there are already other students involved in requirements gathering) and I am hoping for wider participation in the discussion. Since it is a big change, it needs to be coordinated, which is the main point I wanted to make.

**Comment by alanruttenberg** [comment]
To be clear, I don't think these changes should be made now/here. They suggest progress on the IBE/ICE issue that isn't in sync with current thinking. It is an open question as to whether or not to retain a corresponding IBE terms for each ICE term. For example, it is rare to have an IBE chart about which we want to say something interesting. An example of where we might use such a term might be in referring to an original historical chart for which there is one copy. Or if access to specific printed copies of a chart needed to be tracked, though I would expect such tracking to be at a higher level like document.

**Comment by neilotte** [comment]
@alanruttenberg "They suggest progress on X that isn't in line with current thinking, where current thinking considers X an open question" --seems like a recipe for the perfect murdering the good. 

Can you give me an ETA for when your branch will be ready to merge in? We're planning to release a 1.6 just to get this repository looking respectable (maybe in the next month), and then a major 2.0 release where we seek truth and justice and break some things. Would you target the first or the second for this update?

**Comment by avsculley** [comment]
I recently got a new job at SKS and haven't stayed on top of this. But I can set time aside this weekend to make sure that the work we've done is both consistent with the most recent updates to CCO, and also suitable for viewing.

Currently, the ICE/IBA refactor work is too tangled up with work on information structures that we were also doing. We decided to separate them since the ice/iba refactor stuff is more straightforward, and, if the community wants it to be implemented, could be ready much sooner than work on information structures. This became even more apparent in the recent NCOR CCO working group session on information.
________________________________

**Comment by neilotte** [comment]
Thanks @avsculley!

**Comment by alanruttenberg** [comment]
If you are aiming for 1.6 soon then I would make the IBE/ICE changes in 1.7. As I said, I don't think this pull cleans anything up, because the major flaws remain and these changes don't move in the direction changes are going to happen, so they aren't making progress, IMO. I say 1.7 because the change will require several people to review and because we haven't made one of the decisions, namely whether or not to keep all corresponding IBE terms or deprecate most of them. I lean towards the latter. If that many deprecations happens it might be polite to give a longer warning. Which reminds me that we **really** need a discussion list or two. Probably a general questions and discuss list as well as an announce list, for messages like: There's a new release and maybe for the deprecation (though in the latter case requesting comments, if any, get directed to the discuss list). If a mailing list is set up, please make sure it has a public archive. Google groups is easiest. There's a milestone capability in Github. I haven't used it but it might be worth looking into for planning things like this.

**Comment by neilotte** [comment]
@alanruttenberg Certainly agreed on the need for a listserv, foreshadowing, and the rest of it. I'd like to spend some time finding the best options here and not the biggest fan of google groups. Certainly github has a lot of bells and whistles too that we've only scraped the surface of. 

On the matter of this change: I hear you, but I disagree. I think fixing these definitions aids in clarifying that these are not ICEs and that this will be marginally helpful to users in the interim between now and whenever the larger and more important matter is settled.

**Comment by alanruttenberg** [comment]
We agree to disagree. However I believe we aim for consensus. I don't think my claims unreasonable, and I *have* been spending time working in this area.

**Comment by neilotte** [comment]
Hey Alan, can we maybe discuss briefly over the phone? Can be whenever.

**Comment by avsculley** [comment]
I'm still working on the refactor. It should be coming sometime this week.

**Review by johnbeve — CHANGES_REQUESTED** [review]

File: ArtifactOntology.ttl (hunk 1 above)
Comment (johnbeve) [reply ArtifactOntology.ttl:598]: See also the parent definition: 

- Image - An Information Bearing Artifact that is designed to bear some specific Information Content Entity that represents some entity.

An IBE cannot bear an ICE; the range of 'is bearer of' is SDC. Also, the "...Information Content Entity that represents some entity." should just be "...Representational Information Content Entity". 

Suggested revision (not sure what work "specific" is doing so left it out): 

- Image* - An Information Bearing Artifact that is designed to carry some Representational Information Content. 

With that in mind, suggested revision to @neilotte's proposal: 

- Chart* - An Image that is designed to carry some Representational Information Content Entity that is prescribed by some canonical visual format. 

In other words, a chart carries representational content in alignment with some standard for visual communication, e.g. diagrams, graphs, maps, etc.

File: ArtifactOntology.ttl [hunk 2]
```diff
@@ -2441,7 +2442,7 @@ cco:LightingSystem rdf:type owl:Class ;
 ###  http://www.ontologyrepository.com/CommonCoreOntologies/List
 cco:List rdf:type owl:Class ;
          rdfs:subClassOf cco:InformationBearingArtifact ;
-         cco:definition "An Information Bearing Artifact that represents an ordered sequence of values, where the same value may occur more than once."@en ;
+         cco:definition "An Information Bearing Artifact that contains an ordered sequence of values, where the same value may occur more than once."@en ;
```
Comment (johnbeve) [inline ArtifactOntology.ttl:2445]: Following on the CodeList suggested revision: 

- List - An Information Bearing Artifact that consists of one or more Information Bearing Artifacts that carry Information Content Entities and are the subjects of some Sequence Position Ordinality.

File: ArtifactOntology.ttl [hunk 3]
```diff
@@ -4750,7 +4751,7 @@ cco:TurbojetAirBreathingJetEngine rdf:type owl:Class ;
 ###  http://www.ontologyrepository.com/CommonCoreOntologies/TwoDimensionalBarCode
 cco:TwoDimensionalBarCode rdf:type owl:Class ;
                           rdfs:subClassOf cco:Barcode ;
-                          cco:definition "A Barcode whose concretizations consist of two-dimensional symbol patterns."@en ;
+                          cco:definition "A Barcode that consists of a two-dimensional symbol pattern."@en ;
```
Comment (johnbeve) [inline ArtifactOntology.ttl:4754]: Suggested revisions to parent and subclasses:
- Barcode - An Information Bearing Artifact that is designed to bear one or more geometric shapes that concretize some Directive Information Content Entity.

- One-Dimensional Barcode - A Barcode that is designed to bear parallel lines of varying widths and spacing that concretize some Directive Information Content Entity.

- Two-Dimensional Barcode - A Barcode that is designed to bear lines of varying widths, spacing, height, and color that concretize some Directive Information Content Entity.

**Review by johnbeve — CHANGES_REQUESTED** [review]

**Comment by johnbeve** [comment]
I agree with @alanruttenberg on the need for refactoring; I'm not exactly happy with the IBE subclasses...I'm looking eagerly towards @alanruttenberg's and @avsculley's results. 

Even so, I'm inclined to update these IBE definitions (see my proposed revisions to @neilotte's updates) because the original definitions exhibit serious confusions, e.g. an information bearing entity can't represent. I'm worried that these confusions - if left unaddressed - will spread among new users who might be leveraging CCO before the refactoring is implemented. 

With respect to these updates suggesting progress on the IBE/ICE issues that aren't aligned with current thinking, perhaps we can make it clear here and elsewhere that:
- IBE/ICE refactoring is being researched
- These updated classes may not persist through the refactoring
- They're being updated nonetheless because of internal issues with the definitions, independent of the refactoring effort

**Comment by neilotte** [comment]
@johnbeve Ready to review.

File: ArtifactOntology.ttl (hunk 1 above)
Comment (mark-jensen) [reply ArtifactOntology.ttl]: @johnbeve offers:

> Image* - An Information Bearing Artifact that is designed to carry some Representational Information Content.". 

This doesn't distinguish an image from, say, an audio recording. Of course, the original definition wasn't doing that either... 

I understand why the current definition just uses 'represents', since the four main ICEs types are all defined as EqCs based on the specific nature of their aboutness relation to some entity. A representational ICE is just an ICE that represents, as opposed to describes or designates or prescribes. Agree that the use of 'bears' needs to be changed and 'specific' is useless.

Much harder to decide: The differentia for an image has do with it representing a thing because it has an isomorphism with it based on(?) derived from(?) using(?) ....(?) light waves is a visual sense. Os, something like that. Tough stuff. 

If you pushed me to decide today: "An Information Bearing Artifact that is designed to carry an Information Content Entity that represents some entity due to maintaining a visual isomorphism".

Comment (mark-jensen) [reply ArtifactOntology.ttl]: > Chart* - An Image that is designed to carry some Representational Information Content Entity that is prescribed by some canonical visual format.

I like this, but:
1. use of 'prescribed by' means 'canonical visual format' is a Directive ICE, and people will howl that we don't have that in CCO. We'd at least need to add an elucidation that unpacks it with a promise for new additional content some day.
2. I think there needs to be more in the def that reflects that charts are visualizations of underlying data. Something like: "... canonical visual format for representing data". That's gloss, but ... here I go ..

First pass: "An Image that is designed to carry an Information Content Entity that represents a collection of Information Content Entities due to maintaining an isomorphism using some canonical structured visual format."

**Comment by mark-jensen** [comment]
I agree with @alanruttenberg points re making changes here when there is good likelihood that they will be superseded by the much more comprehensive refactor of ICEs and IBA/Es in CCO. However, the timeline to completion and adoption is uncertain. Which makes sense. The refactor is requiring taking a more definitive stance of the nature of GDCs and ICEs. It will be a substantial update to CCO and require a decent amount of vetting by the community. 

These changes reflected in this MR are small improvements, they are needed ones, the work has already mostly been done (see my suggestions), and I think they will transfer nicely into the ongoing refactoring efforts to move IBAs up and over to ICEs. Therefore, I approve them, as short-lived as they may be.

File: ArtifactOntology.ttl (hunk 1 above)
Comment (johnbeve) [reply ArtifactOntology.ttl]: Re Image*: Good catch man. I like shifting to isomorphism talk; how about (to make the isomorphism relata clear): 
- Image** - An Information Bearing Artifact that is designed to carry an Information Content Entity that
represents some entity owing to a visual isomorphism between the artifact and the entity.

Re Chart* and prescribed: Is the concern just that there is not a 'canonical visual format' subclass of Directive ICE? I'd be in favor of adding such a thing in the future (I think this is still independent of the refactoring, i.e. I suspect we'll need it however that goes). 

Re Chart* and restriction to data: I worry there are plausible examples of charts that aren't used to represent data, e.g. @oliviahobai raised the example of an eye chart used in a vision exam. Are, say, organizational charts and flowcharts always used to represent data? Addressing these sorts of examples made me think we should stay broad.

**Review by johnbeve — DISMISSED** [review]

File: ArtifactOntology.ttl (hunk 1 above)
Comment (mark-jensen) [reply ArtifactOntology.ttl]: > Image** - An Information Bearing Artifact that is designed to carry an Information Content Entity that
represents some entity owing to a visual isomorphism between the artifact and the entity

This is good. Suggest small change for maximal clarity "...between the bearing artifact and.."

> I'd be in favor of adding such a thing in the future (I think this is still independent of the refactoring, i.e. I suspect we'll need it however that goes).

Agree on both counts. Neither a deal breaker. 

> ddressing these sorts of examples made me think we should stay broad.

Yeah, I can see that. I am ok with leaving it broad for now. But, I wonder if 'chart' in some cases is a misappropriation of the term. I am pretty sure an Org chart is still representing data. Charts are interesting in that their purpose seems to be in representing information in a way that makes it more amenable to deriving insight. I think that still maintains the isomorphism criteria. Yes? 

Also relevant: the letters in an eye chart seem to be another example of GDCs that aren't about anything. Perhaps the isomorphism is between GDC rather than just ICE.

**Comment by johnbeve** [comment]
@neilotte, I'm on board with @mark-jensen's re-revisions and I believe the revisions overall address @cameronmore's concerns as well. 

Folks on the thread agree refactoring is needed and we're looking forward to seeing what @alanruttenberg and @avsculley have done. The majority seem in favor of making these minor updates while waiting for the more substantial work to come. 

I believe we all agree that these minor updates should **not** suggest to anyone that more substantial revisions to the information content in CCO are undesirable. We're simply revising terms in CCO that exhibit obvious issues while awaiting more substantial revisions. 

So, barring further objections, I'm happy to approve this PR once the latest updates have been made. 

FYSA @APCox @oliviahobai

**Review by mark-jensen — APPROVED** [review]

**Review by mark-jensen — APPROVED** [review]

**Review by mark-jensen — APPROVED** [review]

**Review by mark-jensen — APPROVED** [review]
